roles/installer/ospd/images/build/tasks/main.yml

```diff
@@ -1,2 +1,26 @@
 ---
-# TODO: Add images build code here
+- name: Create overcloud_images directory
+  file: path={{ image_build_path }} state=directory
+
+- name: Download guest image for build
+  get_url:
+      dest: "{{ image_build_path }}/guest_image.qcow2"
+      url: "{{ installer.images.overcloud.guest_image }}"
+
+- name: Build OC images
+  shell: |
+      pushd ~/overcloud_images
+      openstack overcloud image build --all > openstack-build-images.log
```

Comment:
There is no reason to build "all" as we only require the files specified in installer.images.overcloud.files.
Please change this command to: (and verify it works :) )
```
openstack overcloud image build --type {{ item | splitext | first }}
with_items: "{{ installer.images.overcloud.files }}"
```